.gitignore

```diff
@@ -1,3 +1,7 @@
+build/
+install/
+log/
+
 devel/
 bin/
 msg_gen/
```

Comment:
none of these should be in the .gitignore actually (neither your additions, nor what i originally put here). its indicitive of accidentally running colcon commands in the wrong working directory

Reply:
Where should I be running `colcon build`? I've been running it directly in the `uwrt_mars_rover` folder.